src/foam/core/Validation.js

```diff
@@ -402,7 +402,7 @@ foam.CLASS({
       name: 'validationPredicates',
       factory: function() {
         var self = this;
-        const PHONE_NUMBER_REGEX = /^(?:\+?1[-.●]?)?\(?([0-9]{3})\)?[-.●]?([0-9]{3})[-.●]?([0-9]{4})$|^(?:\+?)[5]{2}[-.●]?\(?([0-9]{4})\)?[-.●]?([0-9]{4,5})$/;
```

Comment:
have no idea how to use line continuation here and still be able to use it on backend